### PR TITLE
[skip ci] add race binary

### PR DESCRIPTION
**Pull request by ming535**

### CODE REVIEW

File: Dockerfile.gravity.race [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM frolvlad/alpine-glibc
+
+COPY alpine.repositories /etc/apk/repositories
```
Comment (Ryan-Git) [inline Dockerfile.gravity.race]: we don't need copy this now as it's run on travis

File: Dockerfile.gravity.race [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+RUN apk update && apk upgrade && apk add bash && apk add tzdata
+
+RUN cp /usr/share/zoneinfo/Asia/Shanghai /etc/localtime && echo "Asia/Shanghai" > /etc/timezone
+
+WORKDIR /
```
Comment (Ryan-Git) [inline Dockerfile.gravity.race:7]: the default work dir is the root

File: Makefile [hunk 3]
```diff
@@ -48,11 +48,13 @@ run-dev:
 
 build:
 	$(GOBUILD) -ldflags '$(LDFLAGS)' -o bin/gravity cmd/gravity/single_binary.go
+	$(GOBUILD) -ldflags '$(LDFLAGS)' -race -o bin/gravity-race cmd/gravity/single_binary.go
```
Comment (Ryan-Git) [inline Makefile:51]: where would we use this one?

File: Dockerfile.gravity.race (hunk 2 above)
Comment (ming535) [reply Dockerfile.gravity.race:7]: good to make it explicit.

File: Makefile (hunk 3 above)
Comment (ming535) [reply Makefile:51]: Use it anywhere when it is hard to detect race in test/dev. I will use it on production to re-product a race bug.

**Review by Ryan-Git — APPROVED** [review]

**Pull request merged**